build.py

```diff
@@ -458,8 +458,11 @@
 # TODO: Probably this should be another 'product', but for now it's simply a phase
 foundation_tests = SwiftExecutable('TestFoundation', [
 	'TestFoundation/main.swift',
+        'TestFoundation/HTTPServer.swift',
 ] + glob.glob('./TestFoundation/Test*.swift')) # all TestSomething.swift are considered sources to the test project in the TestFoundation directory
 
+Configuration.current.extra_ld_flags = '-L'+Configuration.current.variables["LIBDISPATCH_BUILD_DIR"]+'/src/.libs'
```

Comment:
@seabaylea :  In build.py, TestFoundation is built as a SwiftExecutable without the linker flags, this results in ld.gold failing to locate libdispatch. I can see that a the rule for SwiftExecutable used EXTRA_LD_FLAGS. Hence added this line. Can you please comment on this?